src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController.swift

```diff
@@ -60,13 +60,9 @@ extension AppInformationViewController {
 		versionLabel.textColor = .enaColor(for: .textPrimary2)
 		versionLabel.style = .footnote
 
-		if let bundleVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"],
-			let bundleBuild = Bundle.main.infoDictionary?["CFBundleVersion"] {
-			versionLabel.text = "\(AppStrings.Home.appInformationVersion) \(bundleVersion) (\(bundleBuild))"
-		} else {
-			versionLabel.text = "\(AppStrings.Home.appInformationVersion) <unknown>"
-			logError(message: "Unknown version. Should not happen!")
-		}
+		let bundleVersion = Bundle.main.appVersion
+		let bundleBuild = Bundle.main.appBuildNumber
+		versionLabel.text = "\(AppStrings.Home.appInformationVersion) \(bundleVersion) (\(bundleBuild))"
```

Comment:
I would question the need for build numbers in the production version. Normally not worth a discussion, People seem to be crazy about the simplest things here. 
I would suggest to omit the build number in prod builds and keep it for the devs and testers available, e.g. compiler flag.

Reply:
Ok I think this is something for the refactoring list. And it also needs to be aligned with whoever had interest to have this in the app.